query/src/configs/config.rs

```diff
@@ -147,6 +148,10 @@ pub struct StoreConfig {
     )]
     #[serde(default)]
     pub rpc_tls_store_service_domain_name: String,
+
+    #[structopt(long, env = "DISABLE_LOCAL_DATABASE_ENGINE", default_value = "0")]
```

Comment:
Look great :heart: 
But `DISABLE_LOCAL_DATABASE_ENGINE` is a `QueryConfig` group item, it used in the query layer.
BTW, the `StoreConfig` group only includes some configs for the store service.

Reply:
OK, I will fix the comment tonight~